people/hiring.md

```diff
@@ -8,6 +8,40 @@ It is meant to be a helpful guide and resources to streamline hiring in the futu
     - We take heavy inspiration from [Project Include's inclusive hiring documentation](https://projectinclude.org/hiring).
 
 
+## Hiring process
+
+Below is a brief overview of the process we must follow any time we wish to make a new job posting.
+
+## Location of information
+
+We have a few resources to help with hiring, see below:
+
+- Our [Hiring Project Board](https://github.com/orgs/2i2c-org/projects/37) tracks all current and past hires, with links to relevant information.
```

Comment:
I have already discussed this board format in my reply to your comment in the gdoc.